src/stream/src/executor/over_window/state/aggregate.rs

```diff
@@ -85,7 +85,6 @@ impl WindowState for AggregateState {
     }
 
     fn curr_output(&self) -> StreamExecutorResult<Datum> {
-        assert!(self.curr_window().is_ready);
```

Comment:
This is OK because `WindowBuffer` allows getting output when current window is not ready (which means the following part of window is not saturated).